<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <meta charset="utf-8" />
    <title>JCollinsMSc_Bioinformatics-DataScience_Resume_171024</title>
    <link href="custom.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="https://platform.linkedin.com/badges/js/profile.js" async defer></script>
</head>

<body id="JCollinsMSc_Bioinformatics-DataScience_Resume_171024" lang="en-US">
    <div style="margin-left: 150px; width: 500px;">
        <div class="_idGenObjectLayout-1">
            <div id="_idContainer011" class="_idGenObjectStyleOverride-2">
                <img class="_idGenObjectAttribute-1" src="image/dna_tree-02.png" alt="" />
            </div>
        </div>
        <div class="underline">
            <div id="_idContainer002">
                <p class="Basic-Paragraph"><span class="Character-Style-19 CharOverride-65">J</span><span class="Character-Style-19">ohn Collins  </span></p>
                <br>
            </div>
            <div>
                <p><span class="CharOverride-66">Python Software Engineering</span></p>
                <p class="Basic-Paragraph ParaOverride-9"><span class="CharOverride-73">Personalized Genomics &amp; Molecular</span></p>
                <p class="Basic-Paragraph ParaOverride-9"><span class="CharOverride-70">Diagnostics | NGS Pipelines | Data Science</span></p>
            </div>
        </div>
        <br>
        <div class="underline">
            <div class="_idGenObjectLayout-1">
                <div id="_idContainer008" class="Basic-Graphics-Frame _idGenObjectStyleOverride-7">
                </div>
            </div>
            <div class="_idGenObjectLayout-1">
                <div id="_idContainer010" class="Basic-Graphics-Frame _idGenObjectStyleOverride-1">
                </div>
            </div>
            <div id="_idContainer001" class="Basic-Graphics-Frame _idGenObjectStyleOverride-2"></div>
        </div>
        <div class="underline">
            <p id="jc-anim-active">BIOINFORMATICS</p>
            <br>
            <p id="underline" class="Paragraph-Style-1 ParaOverride-1"><span class="CharOverride-67-sub">First-hand data analysis on hundreds of next-generation high-throughput sequencing runs</span></p>
            <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-81">Diverse Illumina® expertise (HiSeq™, MiSeq™, NextSeq™)—spanning secondary through tertiary pipeline analysis for computational interpretation of NGS raw sequence data; including, ultra-deep targeted panels, variant calling and genotyping, single-cell, unique molecular barcoding, and transcriptomics.
                </span></p>
        </div>
        <br>
        <br>
        <div class="underline">
            <p class="Paragraph-Style-1 ParaOverride-1"><span class="CharOverride-67-sub">Advanced software development proficiency creating complex custom omics-based analytics using the Python data science stack</span></p>
            <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-81">Daily use of SciPy, NumPy, Pandas, Jupyter, Conda (pkg./virtualenv mgmt.), scikit-learn, matplotlib, in a Linux/UNIX environment; with strong shell scripting, parallel programming, statistics, and machine learning (theano, caffe, keras) skills. Moderate fluency in R and HTML5/CSS3, and working knowledge of C/C++, Java, and JavaScript.
                </span></p>
            <br>
        </div>
        <br>
        <div class="underline">
            <p class="Paragraph-Style-1 ParaOverride-1"><span class="CharOverride-67-sub">Extensive academic and professional history of analysis incorporating myriad open source bioinformatics tools and databases</span></p>
            <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-81">Mapping: BWA(-MEM), Samtools, Bedtools, Bowtie2, STAR; Annotation: UCSC Genome Browser, GENCODE, Ensembl; NCBI: BLAST, RefSeq, SRA, GEO, db-[GaP/SNP/Var], ClinVar/OMIM; Cancer Genomics: TCGA, COSMIC; QC/Other: The Broad Institute—IGV/GATK/Picard/BD2K/LINCS, NIST’s GiaB (NA12878), Platinum genomes; R-Bioconductor; Google Genomics API on GCP; 1000Genomes Project, etc.
            </span></p>
        </div>
        </p>
        <br>
        <div class="underline">
            <div class="_idGenObjectLayout-1">
                <div id="_idContainer003" class="Basic-Graphics-Frame _idGenObjectStyleOverride-3">
                </div>
            </div>
            <br>
            <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-68"> Rapidly improving command</span><span class="CharOverride-81">  of </span><span class="CharOverride-87">state-of-the-art best practices for designing and deploying fast, scalable, high-performance pipelines.</span><span class="CharOverride-81"> Namely, distributed task management for larger-than-memory size datasets (e.g., multi-genomic) using efficient data structures and MapReduce-based cluster computing.<br>
            </span></p>
            <ul>
                <li><b>Cloud computing</b> — cross-platform expertise (Amazon's AWS, MS Azure, GoogleCloudPlatform+GoogleAPIs)
                </li>
                <li><b>Database & Network systems</b> — MySQL, Postgres, Oracle, NoSQL/Cassandra (familiar); ssh tunneling
                </li>
                <li><b>High-speed algorithms design</b> — O(n) execution time complexity⇢efficiency optimization (sw: datatype homogeneity, array vectorization, memory cache⇢data locality, just-in-time compilation, »Cython, Numba, Dask.DataFrame's, Pandas API; hw: »NVIDIA CUDA/GPU’s)
                </li>
                <li><b>Parallelization</b> — concurrent operations (multi-processing, VM-autoscaling); Hadoop/HDFS/Spark functionality in Python via PyData »Blaze, Dask.Distributed, Dask.Parquet, etc. (pandas-based asynchronous task mgmt. & binary serialization; low latency autoscheduling); shell-scripted »GNU Parallel
                </li>
                <li><b>DevOps</b> — version control (»git), unit testing, UI/UX Web/Apps Design (LAMP; »Flask/Django), Agile development practices, containerization (»Docker/Kubernetes)</li>
            </ul>
            </span>
            </p>
        </div>
        <br>
        <hr>
        <br><br>
        <div class="_idGenObjectLayout-1">
            <div id="_idContainer000" class="Basic-Graphics-Frame _idGenObjectStyleOverride-1">
            </div>
        </div>
        <p id="jc-anim-active">EXPERIENCE</p>
        <div id="_idContainer012" class="Basic-Text-Frame">
            <br>
            <p class="Basic-Paragraph ParaOverride-11"><span class="CharOverride-67">Bioinformatician II</span></p>
            <p class="Basic-Paragraph ParaOverride-12"><span class="CharOverride-67-sub">—Sequencing Analysis</span></p>
            <p class="Basic-Paragraph ParaOverride-12"><span class="Character-Style-8 CharOverride-75">  at </span><span class="Character-Style-8 CharOverride-76"><a href="http://www.bio-rad.com/en-us/applications-technologies/digital-pcr-next-generation-sequencing-ngs">Bio-Rad Laboratories</a></span><span class="Character-Style-8 CharOverride-77">,</span><span class="Character-Style-8 CharOverride-75">  </span><span class="Character-Style-8 CharOverride-78"> </span></p>
            <p class="ParaOverride-1"><span class="Character-Style-8 CharOverride-79">Digital Biology Center</span></p>
            <p class="Basic-Paragraph ParaOverride-13"><span class="CharOverride-80">Ap</span><span class="CharOverride-80">ril/May 2017—July 2017</span></p>
            <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-81">Created a working framework for a secondary and tertiary custom RNA-Seq pipeline for a single-cell, whole transcriptome, Unique Molecular Identifier-containing (UMI) assay called ddSEQ™ leveraging Bio-Rad’s Droplet Digital™ technology for individual cell isolation and 3’-end poly(A)-tail bead-capture along with Illumina’s “tagmentation” technology Nextera™  (adapter ligation).</span></p>
            <ul>
                <li class="Basic-Paragraph ParaOverride-15"><span class="CharOverride-87">Developed a cloud computing network architecture</span><span class="CharOverride-97"> </span><span class="CharOverride-94">on </span><span class="CharOverride-99">Microsoft Azure</span><span class="CharOverride-100">, and </span><span class="CharOverride-98">genomically mapped samples in parallel</span><span class="CharOverride-94"> </span><span class="CharOverride-97">u</span><span class="CharOverride-97">sing a </span><span class="CharOverride-100">scalable, distributed cluster of highly multi-core, ephemeral VM instances</span><span class="CharOverride-94">.</span></li>
                <li class="Basic-Paragraph ParaOverride-15"><span class="CharOverride-87">Wrote UMI-barcode deconvolution algorithms</span><span class="CharOverride-90"> </span><span class="CharOverride-100">and integrated database genome-wide “gene” annotations</span><span class="CharOverride-97"> to </span><span class="CharOverride-98">transform reads into digital transcript counts for </span><span class="CharOverride-98">analysis of differential gene expression</span><span class="CharOverride-97">.</span></li>
                <li class="Basic-Paragraph ParaOverride-16"><span class="CharOverride-87">Applied dimensionality reduction,</span><span class="CharOverride-92"> decomposition</span><span class="CharOverride-98">, and</span><span class="CharOverride-92"> feature enrich</span><span class="CharOverride-92">ment</span><span class="CharOverride-100"> </span><span class="CharOverride-98">to quantitatively and visually discriminate human and murine immune system cell subpopulations,</span><span class="CharOverride-97"> from </span><span class="CharOverride-98">Python&apos;s scikit-learn library</span><span class="CharOverride-97">, such as </span><span class="CharOverride-100">K-means clustering, PCA, </span><span class="CharOverride-97">and</span><span class="CharOverride-100"> t-SNE</span><span class="CharOverride-97">.</span></li>
            </ul>
            <br>
            <div class="underline">
                <p class="ParaOverride-1"><span class="CharOverride-67">Bioinformatics Analyst</span><span class="CharOverride-92"> </span></p>
                <p class="ParaOverride-17"><span class="Character-Style-8 CharOverride-75"> </span><span class="Character-Style-8 CharOverride-75"> at </span><span class="Character-Style-8 CharOverride-93"><a href="www.allosure.com">CareDx, Inc. </a></span><span class="Character-Style-8 CharOverride-94">   </span><span class="Character-Style-8 CharOverride-75">     </span></p>
                <p class="ParaOverride-14"><span class="CharOverride-95"> </span><span class="CharOverride-96">(RA/Bioinformatician, 2014-2016</span><span class="CharOverride-96">)</span></p>
                <p class="ParaOverride-13"><span class="CharOverride-80">November 2014—April 2017</span></p>
                <br>
                <center><img src="http://jmd.amjpathol.org/cms/attachment/2069108762/2067529109/gr1_lrg.jpg" width="90%"></center>
                <br>
                <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-81">Python-centric genotyping analysis, rigorous analytical validation, and publication-quality visualization of a deep coverage targeted SNP panel NGS diagnostic that measures “percent donor-derived cell-free DNA” (% dd-cfDNA) without prior genotyping of the donor or recipient, implementing a mix of standard and complex custom software tools with a focus on assay performance characterization.</span></p>
                <ul>
                    <li class="Basic-Paragraph ParaOverride-15"><span class="CharOverride-92">S</span><span class="CharOverride-87">tatistical Python scripting—pipeline vector-graphics reports </span><span class="CharOverride-104">automated to handle large sample input, generating various evolving assay-specific metrics critical for pass/fail classification </span><span class="CharOverride-98">such as </span><span class="CharOverride-100">coverage variability, error, bias, low DNA input (&lt;3ng), heterozygous genotype %’s, </span><span class="CharOverride-98">and </span><span class="CharOverride-100">contamination</span><span class="CharOverride-98">.</span></li>
                    <li class="Basic-Paragraph ParaOverride-15"><span class="CharOverride-92">U</span><span class="CharOverride-87">ncovering sample handling error</span><span class="CharOverride-96"> </span><span class="CharOverride-88">by n</span><span class="CharOverride-101">2</span><span class="CharOverride-88"> (pairwise) </span><span class="CharOverride-102">correlation of allele frequencies</span><span class="CharOverride-93"> </span><span class="CharOverride-104">as a means of </span><span class="CharOverride-105">very</span><span class="CharOverride-104"> high confidence identity verification</span><span class="CharOverride-98"> (visualized by </span><span class="CharOverride-100">hover-interactive hierarchically clustered matrix-coefficient heatmaps </span><span class="CharOverride-106">to reveal correct genomic patient ID→analyte</span><span class="CharOverride-8">).</span></li>
                    <li class="Basic-Paragraph ParaOverride-15"><span class="CharOverride-92">D</span><span class="CharOverride-87">etermination of Limit of Quantification</span><span class="CharOverride-88"> by </span><span class="CharOverride-100">nonlinear least squares</span><span class="CharOverride-107"> </span><span class="CharOverride-100">modeling </span><span class="CharOverride-103">on the </span><span class="CharOverride-108">log-transformed CV&apos;s of analytical validation spike-in replicates </span><span class="CharOverride-103">(measuring 0.2-12% dd-cfDNA).</span></li>
                </ul>
            </div>
            <div class="underline">
                <div id="_idContainer015" class="Basic-Text-Frame">
                    <br>
                    <hr>
                    <br>
                </div>
                <br>
                <div class="_idGenObjectLayout-1">
                    <div id="_idContainer000" class="Basic-Graphics-Frame _idGenObjectStyleOverride-1">
                    </div>
                </div>
                <p id="jc-anim-active">PORTFOLIO</p>
                <br>
                <p class="Paragraph-Style-10 ParaOverride-14"><span class="CharOverride-68">Public sources of Python-built NGS analytics</span><span class="CharOverride-81"> can be seen in the figures to the open-access publication below, describing a novel liquid biopsy-like assay for quantitative surveillance of allograft “genome transplant dynamics.” Published in the </span><span class="CharOverride-109">Association for Molecular Pathology</span><span class="CharOverride-81">’s official journal and selected as the cover issue presented during last year’s annual meeting.</span></p>
                <br>
                <p class="ParaOverride-20"><span class="Character-Style-7 CharOverride-110">PUBLICATIONS</span></p>
                <p class="ParaOverride-10"><span class="CharOverride-111">Association for Molecular Pathology—</span></p>
                <p class="ParaOverride-10"><span class="CharOverride-112">The Journal of Molecular Diagnostics</span></p>
                <p class="ParaOverride-10"><span class="CharOverride-93">November 2016, Cover Feature.</span></p>
                <br>
                <p class="ParaOverride-22"><span class="CharOverride-90">Grskovic, et al.</span><span class="CharOverride-118"> </span></p>
                <p class="ParaOverride-23"><span class="Hyperlink CharOverride-119"><a href="http://linkinghub.elsevier.com/retrieve/pii/S1525157816301386">Validation of a Clinical-Grade Assay to Measure Donor-Derived Cell-Free DNA in Solid Organ Transplant Patients</a></span></p>
                <br>
                <img src="http://jmd.amjpathol.org/cms/attachment/2069108762/2067529105/gr3_lrg.jpg" width="70%">
                <br>
                <br>
                <p class="ParaOverride-21"><span class="CharOverride-49">Fig. 3A.</span><span class="CharOverride-114">  </span><a href="http://allosure.com"><span class="CharOverride-115">AlloSure™</span><span class="CharOverride-116">’s</span></a><span class="CharOverride-117"> </span><span class="CharOverride-49">validated Limit of Blank is precisely </span><span class="CharOverride-49">0.1% — the 95th percentile of 180 “blank” sample results (in this case, meaning </span><span class="CharOverride-49">2</span><span class="CharOverride-49">n genomic ploidy; i.e., in which the total relative fraction of plasma-circulating allogeneic cfDNA has an expected mean value of zero percent</span><span class="CharOverride-49">)</span><span class="CharOverride-49">. </span></p>
                <br>
                <hr>
                <br>
                <div class="_idGenObjectLayout-1">
                    <div id="_idContainer000" class="Basic-Graphics-Frame _idGenObjectStyleOverride-1">
                    </div>
                </div>
                <p id="jc-anim-active">EDUCATION</p>
                <br>
                <p class="ParaOverride-9"><span class="CharOverride-120">University of California</span><span class="CharOverride-121"> </span></p>
                <p class="ParaOverride-9"><span class="CharOverride-122">Santa</span><span class="CharOverride-122"> </span><span class="CharOverride-122">Cruz</span></p>
                <p class="ParaOverride-25"><span class="CharOverride-123">2013—2014</span></p>
                <p class="ParaOverride-25"><span class="CharOverride-42">Master of Science (M.Sc.), </span></p>
                <p class="ParaOverride-25"><span class="CharOverride-124">Biomolecular Engineering </span></p>
                <p class="ParaOverride-25"><span class="CharOverride-124">and</span><span class="CharOverride-125"> </span><span class="CharOverride-124">Bioinformatic</span><span class="CharOverride-124">s</span></p>
                <br>
                <p><span class="CharOverride-123">2009—2013</span></p>
                <p><span class="CharOverride-42">Bachelor of Science (B.S.),</span><span class="CharOverride-126"> </span></p>
                <p><span class="CharOverride-124">Bioengineering </span></p>
                <br>
                <br>
                <div class="_idGenObjectLayout-1">
                    <div id="_idContainer016" class="Basic-Graphics-Frame _idGenObjectStyleOverride-8">
                    </div>
                    <br>
                </div>
                <br>
                <p class="ParaOverride-26"><span class="Character-Style-7 CharOverride-127">CONTACT</span></p>
                <p class="Basic-Paragraph ParaOverride-10"><span class="CharOverride-128">650.808.5179</span></p>
                <p class="Basic-Paragraph ParaOverride-10"><a href="mailto:jcollins.bioinformatics%40gmail.com?subject="><span class="Hyperlink CharOverride-129">bioinformatics@johnsgeno.me</span></a></p>
                <p class="Basic-Paragraph ParaOverride-10"><span class="CharOverride-137">LinkedIn</span><span class="CharOverride-138">:</span><span class="CharOverride-139"> </span><a href="https://linkedin.com/in/johncollins-bioinformatics"><span class="Hyperlink CharOverride-140">/in/johncollins-bioinformatics</span></a></p>
                <p class="Basic-Paragraph ParaOverride-10"><span class="CharOverride-141">San Bruno, CA</span><span class="CharOverride-139"> 94066</span></p>
            </div>
        </div>
        <br>
        <br>
    </div>
    <br>
    <br>
    </div>
    <br>
    <br>
</body>

</html>